elide-core/src/main/java/com/yahoo/elide/jsonapi/models/Relationship.java

```diff
@@ -77,6 +77,7 @@ public Relationship(@JsonProperty("links") Map<String, String> links,
                     if (data.isToOne() && resource == null) {
                         continue;
                     }
+                    // TODO: what if resource==null here?
```

Comment:
@aklish What is the intent here when resource is null?

Reply:
We need a test to find out.

Reply:
Added test and fixed TODO.